GANDLF/config_manager.py

```diff
@@ -618,9 +620,7 @@ def _parseConfig(
             params["model"]["class_list"] = temp_classList.split(",")
         else:
             try:
-                params["model"]["class_list"] = ast.literal_eval(
-                    params["model"]["class_list"]
-                )
+                params["model"]["class_list"] = eval(params["model"]["class_list"])
             except AssertionError:
```

Comment:
Is it correct error type to rise here? AssertionError could happen only in response to failed evaluation of assert statement I think. If so, then it will never be risen

Reply:
Agreed - changed to `logging.error`